src/card_operations/card_pair.h

```diff
@@ -32,20 +32,30 @@
 /*****************************************************************************
  * GLOBAL FUNCTION PROTOTYPES
  *****************************************************************************/
+/**
+ * This function pairs a specific card without handling of retap errors.
+ *
+ * @param card_number The number of the card to be paired. It should be an
+ * integer between 1 and 4.
+ *
+ * @return a uint32_t value, which could be an error status, @ref SW_NO_ERROR
+ * for success. Errors such as wrong family id card and wrong card sequence are
+ * not handled and an error code is returned for the same.
+ */
+uint32_t card_pair_without_retap(uint8_t card_number);
```

Comment:
Let's try to maintain consistency between all card operations. Currently they return status of type `card_error_type_e`. Returning `uint32_t` can be confusing.

This comment is applicable to both functions `card_pair_without_retap` & `card_pair_operation`

Reply:
updated